### ISSUE TITLE
Troubleshooting: multiple successfull attempts

### ISSUE THREAD

**Issue by marcosmarxm** [airbytehq/airbyte#4272]
### Tell us about the documentation you'd like us to add or update.
Some cases the database create two job_id and throws error when launch a sync
[Slack convo](https://airbytehq.slack.com/archives/C01MFR03D5W/p1624272832209300)

```
ERROR Job Scheduler Error
java.lang.IllegalStateException: Job 5966 has multiple successful attempts.
	at com.google.common.base.Preconditions.checkState(Preconditions.java:508) ~[guava-30.1-jre.jar:?]
```

### If applicable, add links to the relevant docs that should be updated
- [Link to relevant docs page](https://docs.airbyte.io)

**Comment by ifokeev** [comment]
Same problem here. My connection can't start a job because of this (?)

**Comment by jrhizor** [comment]
@ifokeev the slack conversation https://airbytehq.slack.com/archives/C01MFR03D5W/p1624272832209300 has info on how to fix. We'll definitely follow up on the root issue.

@marcosmarxm full scheduler logs + a dump of the job attempt db tables would be really helpful debugging this.

**Comment by ifokeev** [comment]
@jrhizor fixed it by deleting the job row. Why this error is preventing synchronization? Is there any way not to stuck jobs because of the error?

**Comment by jrhizor** [comment]
Yeah we should be able to respond to these types of errors more gracefully.

**Comment by marcosmarxm** [comment]
@jrhizor the idea is create a section in troubleshooting area in the website. I'm going to open other to keep track of the problem itself.

**Comment by jrhizor** [comment]
here's the non-documentation ticket: https://github.com/airbytehq/airbyte/issues/4378
